metadata/pom.xml

```diff
@@ -31,6 +31,12 @@
     <name>lightblue-mongo: ${project.groupId}|${project.artifactId}</name>
     <dependencies>
         <dependency>
+            <groupId>com.binarytweed</groupId>
+            <artifactId>quarantining-test-runner</artifactId>
+            <version>0.0.1</version>
```

Comment:
Put quarantining-test-runner dependency in root pom's `dependencyManagement` and remove version from here.